extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/bridge.xml

```diff
@@ -13,7 +13,7 @@
         	<property name="vendor">Philips</property>
         </properties>
         
-        <config-description>
+        <config-description uri="hue:bridge">
```

Comment:
Why is it necessary to define the uri here? Isn't that identical to the uri that is used as a default? Unless there is a good reason to overwrite the default, I would not recommend doing this.